crates/topos/src/config/sequencer.rs

```diff
@@ -0,0 +1,89 @@
+use std::path::{Path, PathBuf};
+
+use crate::components::sequencer::commands::Run;
+use crate::config::Config;
+use figment::{
+    providers::{Format, Serialized, Toml},
+    Figment,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct SequencerConfig {
+    /// SubnetId of the local subnet node, hex encoded 32 bytes starting with 0x
+    pub subnet_id: Option<String>,
```

Comment:
I don't think we need all those values as we will refactor and sanitize the CLI by taking only important params. Same for the TCE

Reply:
I know, but I also don't exactly know yet which we need and which we don't. So I don't see the value in slimming it down now, when we have to go over it again anyway to decide what goes where!

Reply:
As mentioned before, we don't need anything in the config files for now. It is part of the next iteration to add and architecture that. I would prefer to have a blank page to avoid miss leading pre existing argument.

But I let @hadjiszs decide on that.